src/lib/storage.js

```diff
@@ -36,7 +36,7 @@ class Storage extends ScratchStorage {
         this.projectHost = projectHost;
     }
     getProjectGetConfig (projectAsset) {
-        return `${this.projectHost}/internalapi/project/${projectAsset.assetId}/get/`;
+        return `${this.projectHost}/${projectAsset.assetId}`;
```

Comment:
This is confusing.  Shouldn't this be referencing `assetHost` or something or are we calling the project JSON an asset too?